reference/artifacts-with-artifactsrewrite/index.md

```diff
@@ -32,6 +32,23 @@ hal config features edit --artifacts-rewrite true
 
 If using Spinnaker 1.20 or later, support for the standard artifacts UI is enabled by default.
 
+## Migrating from the legacy artifacts UI
```

Comment:
@maggieneterval is "migrating" the right word here? This sounds like you're saying...
* here's what's changed and how to deal with it
* here's how to re-enable legacy

Reply:
Fair point, how do you feel about "Transitioning from the legacy artifacts UI" instead?

Reply:
:100: